library/Ivoz/Provider/Domain/Model/User/UserRepository.php

```diff
@@ -53,8 +53,4 @@ public function findCompanyUsersExcludingIds(
     public function findOneByEmail(
         string $email
     );
-
-    public function findOneByTerminalId(
```

Comment:
I don't see any reference to this method in the PR. If it's actually required to be removed, you should erase it from doctrine repository as well